config/services.php

```diff
@@ -44,4 +45,6 @@
                 'priority' => 10,
             ])
     ;
+    $container->parameters()
+        ->set('tailwind.github_token', '%env(default::GITHUB_TOKEN)%');
```

Comment:
I'm not sure if we should name the env var global like `GITHUB_TOKEN` or be more bundle-specific e.g. `TAILWIND_BUNDLE_GITHUB_TOKEN` to avoid some possible conflicts in env vars. I wonder how much users who already have GITHUB_TOKEN env var will want a different GitHub token value for this bundle. Any thoughts?

Reply:
that might by resolved as checking for `TAILWIND_BUNDLE_GITHUB_TOKEN` first, then fallback to `GITHUB_TOKEN` and then to null if not exists